.nsprc

```diff
@@ -0,0 +1,4 @@
+{
+  "exceptions": [],
+  "exclusions": ["marked"]
```

Comment:
@shamasis I've removed the exclusion, and updated the corresponding system test.